.changelog/4224.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+terminating-gateways: Fix bug where namespace field was not being correctly set on acl policies when using the registration CRD and the registered service
+is in the default namespace and the namespace field is unset
+```
```

Comment:
I think we could simplify this. Careful if accepting this suggested change since the backtick patterns seem to throw it off.

```suggestion
```release-note:bug
terminating-gateways: Fix bug where namespace field was not correctly set on ACL policies if using the `Registration` CRD with the service's namespace unset.
```
```